### ISSUE TITLE
64bit Colour

### ISSUE THREAD

**Issue by OzHawk** [paulscottrobson/eris#28]
This is an improvement request.

The FabGL VGA32 hardware "standard" is capable of 6bit colour.  Is it possible to enable this in Eris?

**Comment by paulscottrobson** [comment]
Yes. It's one of those things that is relatively easy to do but I didn't want to force having 6 bit colour on any platform. It's a quasi-palette system (the palette has to be set up before drawing) but just at present uses RGB.  It's a write to $FF10 to set the palette, and the lower byte is 00000bgr, so it would be easy enough to extend this.

The system can theoretically have 256 different colours, but there's only enough RAM in an ESP32 for 4 bit planes, which can be divided into 2 planes, though they don't really exist physically, it's done by colour mapping to work as a background/foreground system for sprite graphics.
